homeassistant/components/zoneminder/__init__.py

```diff
@@ -26,20 +26,23 @@
 DEFAULT_VERIFY_SSL = True
 DOMAIN = 'zoneminder'
 
+HOST_CONFIG_SCHEMA = vol.Schema({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Optional(CONF_PASSWORD): cv.string,
+    vol.Optional(CONF_PATH, default=DEFAULT_PATH): cv.string,
+    vol.Optional(CONF_PATH_ZMS, default=DEFAULT_PATH_ZMS): cv.string,
+    vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
+    vol.Optional(CONF_USERNAME): cv.string,
+    vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
+})
+
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
-        vol.Required(CONF_HOST): cv.string,
-        vol.Optional(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_PATH, default=DEFAULT_PATH): cv.string,
-        vol.Optional(CONF_PATH_ZMS, default=DEFAULT_PATH_ZMS): cv.string,
-        vol.Optional(CONF_SSL, default=DEFAULT_SSL): cv.boolean,
-        vol.Optional(CONF_USERNAME): cv.string,
-        vol.Optional(CONF_VERIFY_SSL, default=DEFAULT_VERIFY_SSL): cv.boolean,
-    })
+    DOMAIN: cv.ensure_list(HOST_CONFIG_SCHEMA)
```

Comment:
We have to do:
```py
vol.All(
    cv.ensure_list, [HOST_CONFIG_SCHEMA])
```

Reply:
That was there in the first commit: https://github.com/home-assistant/home-assistant/pull/19955/commits/30fd3ae2e0aa9389b71c8096b5efe4d22b2063aa#diff-94253932ccc5baf1e76ea94257d3e91dR40

Even with that it, I get the same behavior :(

Reply:
Aha! It works now!